package/MDAnalysis/analysis/rms.py

```diff
@@ -166,7 +166,9 @@
 import logging
 import warnings
 
+import MDAnalysis as mda
 import MDAnalysis.lib.qcprot as qcp
+import MDAnalysis.analysis.align as align
```

Comment:
Why was this import added? Can we remove it again? This almost certainly creates a circular import.
```suggestion
```